torch/csrc/jit/tensorexpr/tensor.h

```diff
@@ -300,6 +307,20 @@ inline ExprHandle Placeholder::load(const std::vector<T>& args) const {
       new Load(data(), ExprHandleVectorToExprVector(params), new IntImm(1)));
 }
 
+template <typename... Ts>
+inline ExprHandle BufHandle::load(const Ts&... ts) const {
+  std::vector<ExprHandle> params({ExprHandle(ts)...});
+  return ExprHandle(
+      new Load(node(), ExprHandleVectorToExprVector(params), new IntImm(1)));
```

Comment:
Nit: you probably could use `Load::make(*this, params)` here to avoid converting from handles to raw exprs and back.